follow_up.yaml

```diff
@@ -0,0 +1,110 @@
+schema: "http://json-schema.org/draft-04/schema#"
+
+id: "follow_up"
+title: Follow Up
+type: object
+namespace: http://gdc.nci.nih.gov
+category: clinical
+program: '*'
+project: '*'
+description: >
+  A visit by a patient or study participant to a medical professional. A clinical encounter that
+  encompasses planned and unplanned trial interventions, procedures and assessments that may be
+  performed on a subject. A visit has a start and an end, each described with a rule. The process
+  by which information about the health status of an individual is obtained before and after a study
+  has officially closed; an activity that continues something that has already begun or that repeats
+  something that has already been done.
+additionalProperties: false
+submittable: true
+validators: null
+
+systemProperties:
+  - id
+  - project_id
+  - state
+  - created_datetime
+  - updated_datetime
+
+links:
+  - name: subjects
+    backref: follow_ups
+    label: describes
+    target_type: subject
+    multiplicity: many_to_one
+    required: true
+
+required:
+  - submitter_id
+  - type
+  - subjects
+
+uniqueKeys:
+  - [id]
+  - [project_id, submitter_id]
+
+properties:
+  $ref: "_definitions.yaml#/ubiquitous_properties"
+
+  offset:
+    description: >
+      number of days since the start of symptoms or hospitalization for each image, this is very important to have when there are multiple images for the same patient to track progression while being imaged. If a report says "after a few days" let's assume 5 days.
+    type: integer
+
+  intubation_present:
+    description: >
+      Yes (Y) or no (N) or blank if unknown
+    enum:
+      - "Y"
+      - "N"
+
+  in_icu:
+    description: >
+      Yes (Y) if the patient is in the ICU (intensive care unit) or CCU (critical care unit) or No (N) or blank if unknown.
+    enum:
+      - "Y"
+      - "N"
+
+  extubated:
+    description: >
+      Yes (Y) if the patient was successfully extubated or No (N) or blank if unknown
+    enum:
+      - "Y"
+      - "N"
+
+  temperature:
+    description: >
+      Temperature of the patient in Celsius at the time of the image
+    type: number
+
+  pO2_saturation:
+    description: >
+      partial pressure of oxygen saturation in % at the time of the image
+    type: number
+
+  leukocyte_count:
+    description: >
+      white blood cell count in units of 10^3/uL at the time of the image
+    type: number
+
+  neutrophil_count:
+    description: >
+      neutrophil cell count in units of 10^3/uL at the time of the image
+    type: number
+
+  lymphocyte_count:
+    description: >
+      lymphocyte cell count in units of 10^3/uL at the time of the image
+    type: number
+
+  view:
+    description: >
+      Posteroanterior (PA), Anteroposterior (AP), AP Supine (APS), or Lateral (L) for X-rays; Axial or Coronal for CT scans
+    type: string
+
+  modality:
+    description: >
+      CT, X-ray, or other
+    type: string
```

Comment:
Change description to "Medical imaging modalities, for example CT Scan, X-ray, or MRI.

Reply:
done